mappings/net/minecraft/client/particle/ItemPickupParticle.mapping

```diff
@@ -1,6 +1,20 @@
 CLASS net/minecraft/class_693 net/minecraft/client/particle/ItemPickupParticle
-	FIELD field_20944 bufferBuilderStorage Lnet/minecraft/class_4599;
-	FIELD field_3821 picker Lnet/minecraft/class_1297;
-	FIELD field_3823 item Lnet/minecraft/class_1297;
-	FIELD field_3824 entityRenderDispatcher Lnet/minecraft/class_898;
-	FIELD field_3826 existingTicks I
+	FIELD field_20944 bufferStorage Lnet/minecraft/class_4599;
+	FIELD field_3821 interactingEntity Lnet/minecraft/class_1297;
+	FIELD field_3823 itemEntity Lnet/minecraft/class_1297;
+	FIELD field_3824 dispatcher Lnet/minecraft/class_898;
+  FIELD field_3826 ticksExisted I
```

Comment:
```suggestion
```
Need to fix this build failure before merge. The same line already exists below.

Reply:
Oh, whoops. That was a mismergehap.